bundles/org.openhab.binding.flicbutton/README.md

```diff
@@ -0,0 +1,119 @@
+  as they're addded with [simpleclient](https://github.com/50ButtonsEach/fliclib-linux-hci). If they're already
+  attached to the flicd-bridge before configuring this binding, they can be discovered by triggering an active scan.
+
+## Thing Configuration
+
+### flicd-bridge
+
+Example for textual configuration:
+
+```
+Bridge flicbutton:flicd-bridge:mybridge
+```
+
+The default host is localhost:5551 (this should be sufficient if flicd is running with default settings on the same
+server as openHAB). If your flicd service is running somewhere else, specify it like this:
+
+```
+Bridge flicbutton:flicd-bridge:mybridge [ hostname="<YOUR_HOSTNAME>",  port=<YOUR_PORT>]
+```
+
+If flicd is running on a remote host, please do not forget to start it with the parameter `-s <openHAB IP>`, otherwise it won't be accessible for openHAB (more details on [fliclib-linux-hci](https://github.com/50ButtonsEach/fliclib-linux-hci)).
+
+### button
+
+For the button, the only config parameter is the MAC address. Normally, no textual configuration is necessary as buttons are autodiscovered as soon as the bridge is configured. If you want to use textual configuration anyway, you can do it like this:
+
+```
+Bridge flicbutton:flicd-bridge:mybridge [ hostname="<YOUR_HOSTNAME>",  port=<YOUR_PORT>] {
+    Thing button myflic1 "<YOUR_LABEL>" [address ="<MAC_ADDRESS>"]
+    Thing button myflic2 "<YOUR_LABEL>" [address ="<MAC_ADDRESS>"]
+    ...
+}
+```
+
+You can lookup the MAC addresses of your buttons within the inbox of the UI. You're free to choose any label you like for your button.
+
+## Channels
+
+| Channel ID                | Channel Type             | Item Type                | Description                    |
+| ------------------------- | ------------------------ | --------------------------| ------------------------------ |
+| rawbutton                | [System Trigger Channel](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-trigger-channel-types) `system.rawbutton`  | Depends on the  [Trigger Profile](https://www.openhab.org/docs/configuration/items.html#profiles) used | Raw Button channel that triggers `PRESSED` and `RELEASED` events, allows to use openHAB profiles or own implementations via rules to detect e.g. double clicks, long presses etc.  |
+| button                   | [System Trigger Channel](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-trigger-channel-types) `system.button`    | Depends on the [Trigger Profile](https://www.openhab.org/docs/configuration/items.html#profiles) used | Button channel that is using Flic's implementation for detecting long, short or double clicks. Triggers `SHORT_PRESSED`,`DOUBLE_PRESSED` and `LONG_PRESSED` events.   |
+| battery-level            | [System State Channel](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types) `system.battery-level`     | Number | Represents the battery level as a percentage (0-100%).
+
+
```

Comment:
```suggestion

```
I believe the code checker will flag double empty lines, changing to use only one.